app/Models/Signup.php

```diff
@@ -131,10 +133,26 @@ public function getQuantity()
         // logic to store everything in the quanity column and not use the quanity_pending
         // column at all. We only want to return what is in the quanity_pending column
         // if is the only place quanity is stored.
-        if (! is_null($this->quantity_pending) && is_null($this->quantity)) {
-            return $this->quantity_pending;
+
+        // If the there is at least one post and it has NULL quantity,
+        // it means that the quantity is still on the signup.
+        $posts = $this->posts;
+
+        if ($posts->count() <= 0 || is_null($posts->first()->quantity)) {
```

Comment:
Nitty gritty but this could probably be `== 0` or could use `isEmpty()` to determine if there are posts or not

Reply:
ah yes - I like `isEmpty()` a lot better!